XamarinCommunityToolkit/Behaviors/MaxLengthReachedBehavior.shared.cs

```diff
@@ -0,0 +1,51 @@
+﻿using System;
+using System.ComponentModel;
+using System.Windows.Input;
+using Xamarin.Forms;
+
+namespace Xamarin.CommunityToolkit.Behaviors
+{
+    public class MaxLengthReachedBehavior : BaseBehavior<InputView>
```

Comment:
Can we change this name? Because we can have something similar to label as well. Something like, when the text in the labels reaches out the limit we add "..." at the end of the string. You can see this feature [here](https://github.com/kphillpotts/ArtAuction/blob/master/src/ArtAuction/ArtAuction/CustomControl/LabelLength.xaml.cs), that time I did a custom control, but I think that it can be converted into a `behavior`.

Reply:
Hi @pictos , I like your suggestion! But the 'behavior' of your suggestion is something totally different then this one and I don't think we should combine this into 1 behavior. Because that would become confusing in my opinion that the behavior would behave differently when set on an Entry than it would behave on a Label. But your suggestion could lead to something like a `MaxLengthFormatBehavior`? And I would love to implement this, if you would like me to implement this as well (in a different PR)? Now I think of it, wouldn't a ValueConverter be a better option for your scenario? What do you guys think @jfversluis, @AndreiMisiukevich, @VladislavAntonyuk?

Reply:
@jBijsterboschNL I'm not suggesting you implement on this PR, I'm just pointing out that this name can lead to some confusion in our users, since at least to me, it can apply in every text control that has a Text prop(editor, entry, label, button, etc) and isn't true, since this implementation is just for `InputView`.

> And I would love to implement this, if you would like me to implement this as well (in a different PR)?

Go for it! I'll love to see it!

> Now I think of it, wouldn't a ValueConverter be a better option for your scenario? What do you guys think @jfversluis, @AndreiMisiukevich, @VladislavAntonyuk?

Also, we can create an issue later on to discuss the specification.

Reply:
I think we can leave as is for now

Reply:
@AndreiMisiukevich are you saying about the name? If it takes into consideration that a name change is a breaking change

Reply:
I mean this behavior triggers a command when maximum length is reached (also it doesn't allow to add more characters)

Why is this name not good?

```MaximumLengthReachedBehavior``` ?

Reply:
Also all other 'read-only' controls with the MaxLength property cannot 'reach' the max length by any user input. That would clarify why this Behavior is not available for the Button and Label controls (and also explains the name for this behavior, at least for me). Also for those 'read-only' controls the actual behavior would be different (not invoking an EventHandler or executing a Command). @pictos what name would you propose then?

Reply:
@jBijsterboschNL I'm terrible with names, I was hoping that you came with a great name xD. Maybe `InputMaxLengthReachedBehavior` or `InputMaxLengthBehavior`, I think that having an `Input` prefix in the name will be more clear the usage of this behavior.
But I'm ok with keeping this original name if this makes sense to you.

Reply:
Hey guys, please reach an agreement regarding the final name